src/modules/0.0.14/aws_vpc/index.ts

```diff
@@ -543,7 +567,11 @@ export const AwsVpcModule: Module2 = new Module2({
               await AwsVpcModule.mappers.vpc.db.update(cloudRecord, ctx);
               out.push(cloudRecord);
             } else {
-              await AwsVpcModule.mappers.vpc.cloud.create(e, ctx);
+              const res = await AwsVpcModule.mappers.vpc.cloud.create(e, ctx);
+              if (!AwsVpcModule.utils.eqTags(e.tags, cloudRecord.tags) && e.vpcId && e.tags) {
+                await updateTags(client.ec2client, res.vpcId, e.tags);
+              }
+
```

Comment:
oh wait you should not be doing this here, this is the replace path. Also, please add the `updateOrReplace` mapper function to identify the cases (we forgot in the previous PRs). Then, the tags' update and the current restore should go through the `update` path.